docs/pages/User/GettingStarted/peer.md

```diff
@@ -12,21 +12,33 @@ Liqo can automatically discover any available clusters or make any clusters disc
 Using kubectl, you can also manually obtain the list of discovered foreign clusters:
 
 ```bash
-kubectl get foreignclusters
-NAME                                   AGE
-ff5aa14a-dd6e-4fd2-80fe-eaecb827cf55   101m
+kubectl get foreignclusters.discovery.liqo.io
```

Comment:
Why are those resources associated to the "discovery" object?
Peering happens *before* discovery.